auth0_flutter/example/ios/Tests/WebAuth/WebAuthHandlerTests.swift

```diff
@@ -121,7 +125,7 @@ extension WebAuthHandlerTests {
                 let result = WebAuthHandler().methodHandlerProvider(method, client)
                 XCTAssertTrue(type(of: result) == methodHandler)
                 expectation.fulfill()
-                return result
+                return mockMethodHandlers[method]!
```

Comment:
Otherwise, the login popup will show up when running the tests on macOS. Since the iOS tests run inside a simulator, it wasn't really relevant until now.